substrafl/strategies/fed_avg.py

```diff
@@ -96,12 +100,14 @@ def perform_round(
                 current_aggregation=None,
                 round_idx=0,
                 aggregation_id=aggregation_node.organization_id,
+                clean_models=clean_models,
             )
 
         current_aggregation = aggregation_node.update_states(
             self.avg_shared_states(shared_states=self._shared_states, _algo_name="Aggregating"),  # type: ignore
             round_idx=round_idx,
             authorized_ids=list(set([train_data_node.organization_id for train_data_node in train_data_nodes])),
+            transient_outputs=clean_models,
```

Comment:
why is it `transient_ouputs` here and `clean_model` in `_perform_local_updates` ?
If we want to expose `clean_model` to the end user, I think it's better to use `clean_model` everywhere, except for the substra API, as some users will write their own strategies

Reply:
It made more sense to me as I understood from @louishulot that Strategies were not written by end users but if that's the case then yes I can standardise.

Reply:
I think the goal was to make this possible in the future, plus these functions are visible in the documentation so I think it's better to stay consistent, to not confuse the users and also the devs.